src/main/features/claim/routes/amount.ts

```diff
@@ -22,6 +30,10 @@ export default express.Router()
     ErrorHandling.apply(async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
       const form: Form<Amount> = req.body
 
+      if (form.model.lowerValue != null && form.model.lowerValue < 0.01) {
+        addErrorMessage(form, 'lowerValue', ValidationErrors.LOWER_VALUE_AMOUNT_NOT_VALID)
```

Comment:
why not add a min validator to the model?

Reply:
min makes it mandatory

Reply:
Can't you just add an `@IsOptional()`

Reply:
@IsOptional() just check for null